app/web/src/features/auth/Settings.tsx

```diff
@@ -18,21 +19,29 @@ import {
 } from "./constants";
 import useAccountInfo from "./useAccountInfo";
 
+const usePageTitleStyles = makeStyles((theme) => ({
```

Comment:
We usually just call it `useStyles` if it's the styles only for this file/component (and then change `root` to `pageTitle`) - but not a big deal

Reply:
Changed it in last push